src/docs/software/using/python.md

```diff
@@ -0,0 +1,116 @@
+Python is an interpreted, object-oriented, high-level programming language with dynamic semantics.
+Its high-level built in data structures, combined with dynamic typing and dynamic binding, 
+make it very attractive for Rapid Application Development, as well as for use as a scripting or 
+glue language to connect existing components together. Python's simple, easy to learn syntax 
+emphasizes readability and therefore reduces the cost of program maintenance. 
+
+
+## Python versions
+Sherlock features two different versions of Python: 2.7 and 3.6.1. 
+The reason for the two versions is that some code, or modules, will only work with old features 
+of version 2 while most code with work on version 3. 
+The general consensus is to use Python 3 whenever you can, especially for most numerical based modules 
+(for example: numpy, scipy, tensorflow, keras, etc).
+
+You can load either version on Sherlock by doing the following:
+ 
+```
+$ ml python/3.6.1
+```
+For Python 3
+
+or
+     
+```
+$ ml python/2.7.13
+```
+
+If just load "python" without a version number, it will default to Python 2.7.
+
+
+## Installing Python modules in your environment
```

Comment:
The website layout is not so great with longer section titles. So maybe just something like:
`## Installing additional Python packages`?